src/styles/main.scss

```
@@ -1 +1,79 @@
-// put code here
+$watch-size: 500px;
+$watch-center-size: $watch-size * 0.05;
+$watch-minute-hand-size: $watch-size * 0.03;
+$watch-second-hand-size: $watch-minute-hand-size * 0.3;
+$main-color: black;
+$secondary-color: green;
+$minute-color: blue;
+$center-color: yellow;
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.watch {
+  position: relative;
+  display: flex;
+  height: $watch-size;
+  width: $watch-size;
+  border: 8px dashed $main-color;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    z-index: 2;
+    left: ($watch-size - $watch-center-size) / 2;
+    top: ($watch-size - $watch-center-size) / 2;
+    height: $watch-center-size;
+    width: $watch-center-size;
+    background-color: $center-color;
+    border-radius: 50%;
+  }
+
+  &__minute-hand {
+    position: absolute;
+    z-index: 1;
+    top: ($watch-size - $watch-minute-hand-size) / 4;
+    left: ($watch-size - $watch-minute-hand-size) / 2;
+    height: $watch-size * 0.25;
+    width: $watch-minute-hand-size;
+    background-color: $minute-color;
+    border-radius: 5px;
+
+    animation: move;
+    animation-duration: 60s;
+    animation-timing-function: linear;
+
+  }
+
+  &__second-hand {
+    position: absolute;
+    left: ($watch-size - $watch-second-hand-size) / 2;
+    top: 0;
+    height: $watch-size * 0.5;
+    width: $watch-second-hand-size;
+    background-color: $secondary-color;
+    border-radius: 5px;
+
+    animation-name: move;
+    animation-duration: 5s;
+    animation-timing-function: steps(60);
+    animation-iteration-count: 12;
+
```

```suggestion
    animation-name: move;
    animation-duration: 5s;
    animation-timing-function: linear;
    animation-iteration-count: infinite;
```
this will make move mopre smooth